src/sentry/static/sentry/app/components/organizations/headerItem.jsx

```diff
@@ -53,30 +58,58 @@ class HeaderItem extends React.Component {
         className={className}
         isOpen={isOpen}
         hasSelected={hasSelected}
+        locked={locked}
         {...props}
       >
-        <IconContainer hasSelected={hasSelected}>{icon}</IconContainer>
+        <IconContainer hasSelected={hasSelected} locked={locked}>
+          {icon}
+        </IconContainer>
         <Content>{children}</Content>
         {hasSelected &&
-          allowClear && <StyledClose src="icon-close" onClick={this.handleClear} />}
-        <StyledChevron
-          isOpen={isOpen}
-          hasChanges={hasChanges}
-          onClick={this.handleChevronClick}
-        >
-          <InlineSvg src="icon-chevron-down" />
-        </StyledChevron>
+          !locked &&
+          allowClear && (
+            <StyledClose
+              src="icon-close"
+              locked={locked}
+              hasSelected={hasSelected}
+              onClick={this.handleClear}
+            />
+          )}
+        {!locked && (
+          <StyledChevron
+            isOpen={isOpen}
+            hasChanges={hasChanges}
+            onClick={this.handleChevronClick}
+          >
+            <InlineSvg src="icon-chevron-down" />
+          </StyledChevron>
+        )}
+        {locked && (
+          <Tooltip
+            title={lockedMessage || 'This selection is locked'}
```

Comment:
Is 'This selection cannot be modified' clearer on what the lock is preventing? :golf:

Reply:
Any suggestions for a better default?

Reply:
`The current view does not allow any changes`?